components/Results.tsx

```diff
@@ -1,53 +1,37 @@
-import styles from '@/styles/Attendance.module.css';
-import { ExamData } from '@/types/Results';
+import styles from "@/styles/Attendance.module.css";
+import { ExamData } from "@/types/Results";
 
-import Skeleton from 'react-loading-skeleton';
-import 'react-loading-skeleton/dist/skeleton.css';
-import ResultsCard from './cards/ResultsCard';
+import Skeleton from "react-loading-skeleton";
+import "react-loading-skeleton/dist/skeleton.css";
+import ResultsCard from "./cards/ResultsCard";
 
 const ResultsTable = ({ data }: { data: ExamData | null }) => {
   return (
     <>
-      <table style={{ width: '-webkit-fill-available' }}>
+      <table className="w-full">
         <thead />
         <thead />
-        <tbody className={[styles.attr, 'attTable'].join(' ')}>
+        <tbody className={[styles.attr, "attTable"].join(" ")}>
           {data && data.marks[0] ? (
             data.marks.map((element, index: number) => (
               <ResultsCard key={index} data={element} />
             ))
           ) : data?.name ? (
-            <div
-              style={{
-                background: 'var(--background-light)',
-                borderRadius: '22px',
-                height: '280px',
-                display: 'flex',
-                alignItems: 'center',
-                justifyContent: 'center',
-                margin: '12px 24px',
-              }}
-            >
-              <h4
-                style={{
-                  textAlign: 'center',
-                  opacity: '0.8',
-                  fontSize: '16px',
-                }}
-              >
+            <div className="m-[12px_24px] flex h-[280px] items-center justify-center rounded-[22px] bg-backgroundLight">
+              <h4 className="text-center text-[16px] opacity-80">
```

Comment:
start using text-md text-sm and so on instead of literally text-[16px], this makes site much more responsive as it uses rem (relative unit) instead of px (absolute unit)